src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar__input {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
The `font-family` for `.searchbar__input` should consistently include fallback fonts. Ensure that `Arial`, `Helvetica`, or similar are included as fallbacks.